lib/stringify/source-map-support.js

```diff
@@ -6,7 +6,11 @@
 var SourceMap = require('source-map').SourceMapGenerator;
 var SourceMapConsumer = require('source-map').SourceMapConsumer;
 var sourceMapResolve = require('source-map-resolve');
-var fs = require('fs');
+try {
+  var fs = require("fs");
+} catch (e) {
+  var fs = null;
```

Comment:
There should be check for `fs !== null` in all methods, that depends on it. In case `fs` is not loaded it should throw some error to make it clear.

Reply:
I think I've done that - there's only one place that needs changing for runtime I think?

Reply:
I've now created a second PR with an alternative approach - I think I like it a little more - but up to you! 😄 

https://github.com/reworkcss/css/pull/147

Reply:
I'm not sure to understand but if it's a dep, why first to not add it to the dependencies?

Reply:
I have no idea what that means. But with this workaround I can parse CSS in the browser... Meaning I get to use the jasmine-dom library from Karma! 😍 So, what needs to change? 🙏